index.html

```diff
@@ -235,6 +235,22 @@ <h3 class="separator">Skills</h3>
                     </ul>
                 </div>
             </section>
+            <section class="contact-me">
+                <div>
+                    <h3>Contact me</h3>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                        <br>I’d love to help with it!
+                    </p>
+                </div>
+                <form action="https://formspree.io/f/{form_id}" method="post"></form>
+                    <ul>
+                        <li><input type="text" name="text" id="text" maxlength="30" placeholder="Your Full Name" required></li>
+                        <li><input type="email" name="email" id="email" placeholder="Your Email ID" maxlength="30" required></li>
```

Comment:
- [ ] please remove the `maxlength` attribute from the email Input field. it is only required in input type text and textarea fields